routes/api/bookin.js

```diff
@@ -365,7 +365,7 @@ router.post('/changeReservation',(req,res)=>{
 // @desc cancel a reservation 
 // @access public
 router.post('/cancel', (req,res)=>{
-    Booking.findById(req.query.bookingID)
+    Booking.findById(req.body.bookingID)
```

Comment:
I consulted the backend, to make sure they are ok with the change. It is because I somehow cannot get the req.query object with a POST request. Also, POST requests and req.body is the more conventional way to go.